src/sentry/shared_integrations/exceptions/base.py

```diff
@@ -0,0 +1,41 @@
+from collections import OrderedDict
+
+from bs4 import BeautifulSoup
```

Comment:
What a quirky name lol, didn't know this existed

Reply:
BeautifulSoup is the best HTML parser, especially for one-off scripts. https://www.crummy.com/software/BeautifulSoup/